SYNC_UPSTREAM.md

```diff
@@ -205,58 +202,69 @@ bin/hcc `bin/hcc-config --build --cxxflags --ldflags` \
 ./a.out ; echo $?
 ```
 
-### Push HCC Clang submodule
+### Push and Create Pull Request for HCC Clang submodule
 
 - change to HCC directory
 - `cd clang`
-- `git checkout clang_tot_upgrade`
-- `git push`
+- `git push clang_fork merge_20170626:merge_20170626`
```

Comment:
@aaronenyeshi there are date-specific information here. could you make it more general?